.tool-versions

```diff
@@ -1,5 +1,5 @@
 # this file is used by asdf to automatically select the correct tool versions
 # but it also serves as a reference for non-asdf users
 python 3.11.4
-terraform 0.13.7
+terraform 1.7.4
```

Comment:
For the time being it may be better to use something in 1.5.x for the license stuff, just in case. Unless there's a very compelling reason to have 1.7.4

Reply:
https://github.com/app-sre/container-images/pull/98